### ISSUE TITLE
Question: "Some diagram description contains errors"

### ISSUE THREAD

**Issue by tomer-ben-david** [skuro/plantuml-mode#71]
# Summary

When I generate with `C-c C-c` (or any other way) I get on the buffer where the image should present:

```
ERROR
0
Empty description
Some diagram description contains errors
```

note that generating the image is fine with: `C-c C-e` then export to html.

> Write here the summary of your changes

# Checklist for pull requests

- [ ] I'm fine with my code being released under the GPLv2 license
- [ ] the pull request is filed against the `develop` branch instead of `master`

**Comment by fingon** [comment]
I am encountering the same thing as well. Not huge problem as I can have plantuml running in different window to see the document as I edit it but still.

**Comment by alphatan** [comment]
Are you running plantuml with the version **plantuml.1.2018.11.jar** and parameter `-config some.config` ?

It seems a bug of the specific version of plantuml. You may downgrade the jar to **plantuml.1.2018.10.jar** or previous ones.

FYI.

**Comment by asbjornu** [comment]
I am experiencing this error now and can't for the life of me figure out what is wrong in the diagram. Would it be possible for the error message to be a bit more specific? "Some diagram description contains errors" doesn't say much. I'm using `-pipe` and are only feeding `plantuml.jar` one diagram to convert, which looks like this:

```plantuml
@startuml Diagram
actor client
node app
database db
db -> app
app -> client
@enduml
```

**Comment by asbjornu** [comment]
By building PlantUML from source and running the tests, I discovered that plantuml/plantuml#600 fixed my issue, which pointed to excessive whitspeace after `@enduml` being the source of my problem. Simply trimming the whitespace made everything work.

**Comment by schmr** [comment]
> excessive whitspeace after `@enduml` being the source of my problem. Simply trimming the whitespace made everything work.

Thanks @asbjornu for reporting it here, saved me the time when I got the same problem!

**Comment by bloggins666** [comment]
Hello,

I'm on a ubuntu 22.04LTS server and I'm running wiki media very recent release (I can't find the #$$$@! version tag!!)

I followed the installation for the Diagram extension and followed the instructions for adding it to my LocalSettings.php file.

I added the following code to the bottom of a wki page I had created...

` <plantuml> @startuml <your PlantUML code here> @enduml </plantuml>`

Which is exactly what I get on the rendered wiki page.

And now I get this:
> Diagrams error (with plantuml command): Error line 2 in file: /tmp/diagrams_inc203710de967.plantuml Some diagram description contains errors

Any thoughts?
